addons/project_sale_expense/models/project_project.py

```diff
@@ -12,7 +12,7 @@ class Project(models.Model):
 
     def _get_expenses_profitability_items(self, with_action=True):
         expenses_read_group = self.env['hr.expense']._read_group(
-            [('state', 'in', ['approved', 'done']), ('analytic_distribution', 'in', self.analytic_account_id.ids)],
+            [('sheet_id.state', 'in', ['post', 'done']), ('analytic_distribution', 'in', self.analytic_account_id.ids)],
```

Comment:
Same comment than the first one I made in that review.

Is your change not equivalent to the previous one?

We will maybe have more records with the previous one but I guess we will have the same result in `Billed` column, no?